src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/repositories/FermaFlowPathRepository.java

```diff
@@ -151,6 +151,17 @@ public Optional<FlowPath> findByCookie(FlowSegmentCookie cookie) {
         return flowPathFrames.isEmpty() ? Optional.empty() : Optional.of(flowPathFrames.get(0)).map(FlowPath::new);
     }
 
+    @Override
+    public Collection<PathId> findPathIdsBySharedBandwidthGroupId(String sharedBandwidthGroupId) {
+        return framedGraph().traverse(g -> g.V()
+                        .hasLabel(FlowPathFrame.FRAME_LABEL)
+                        .has(FlowPathFrame.SHARED_BANDWIDTH_GROUP_ID_PROPERTY, sharedBandwidthGroupId)
+                        .values(FlowPathFrame.PATH_ID_PROPERTY))
+                .getRawTraversal().toStream()
+                .map(pathId -> PathIdConverter.INSTANCE.toEntityAttribute((String) pathId))
+                .collect(Collectors.toList());
+    }
+
```

Comment:
Could you please add unit test for this method?

Reply:
Done